Monika After Story/game/script-topics.rpy

```diff
@@ -4299,6 +4299,28 @@ label monika_write:
     m "Thanks for listening~"
     return
 
+init 5 python:
+      addEvent(Event(persistent.event_database,eventlabel="monika_writingtip4",category=['writing tips'],prompt="Writing tip #4",pool=True))
+  
+label monika_writingtip4:
+     m 3b "Here's Monika's Writing Tip of the Day!"
+     m "You know about writer's block, right?"
+     m 1e "I had it a lot, when I started writing."
+     m "Sometimes it was halfway through a draft, but more often before I even started."
+     m 2f "Every time I tried to write a word, I thought, 'this isn't going to sound good', or 'this isn't how I want it to look'. So I'd stop, backtrack, and try again."
+     m 2d "But I realised it ultimately didn't matter if things didn't work out first time!"
+     m 1a "I feel the heart of writing is not about getting it right first time, but about perfecting it afterwards."
```

Comment:
m 2a "I feel the heart of writing is not about getting it right first time, but about perfecting it afterwards."